src/components/DefaultHeader.js

```diff
@@ -1,22 +1,23 @@
 import React, { useCallback } from 'react'
 import PropTypes from 'prop-types'
 import useHeaderCell from '../hooks/useHeaderCell'
+import { ASCENDING, DESCENDING } from '../constants/sortDirections'
 
 const SimpleHeader = ({ columnIndex }) => {
-  const { setSorts, id, label, columnSort, sortable } = useHeaderCell(columnIndex)
+  const { setSorts, id, label, sortDirection, sortable } = useHeaderCell(columnIndex)
 
   const handleSort = useCallback(() => {
     if (!sortable) {
       return
     }
 
-    const isAsc = columnSort ? !columnSort.isAsc : true
+    const nextDirection = sortDirection === ASCENDING ? DESCENDING : ASCENDING
 
-    setSorts(id, isAsc)
-  }, [setSorts, id, columnSort])
+    setSorts(id, nextDirection)
```

Comment:
I would like to move the sort handler into the hook and return `setSortDirection(direction<string>)` instead of `setSorts`.  This will make the hook easier to use and allow us to add additional shared logic like setting a maximum number of sorts.